q2_vsearch/plugin_setup.py

```diff
@@ -404,12 +406,17 @@
                   'abundances).'),
     },
     parameter_descriptions={
+        'method': ('Denovo chimera detection based on uchime (Edgar 2011), '
```

Comment:
@colinvwood While trying to keep this short and sweet, I've added a little more detail.

How does this look?

I think we should cite Rob's papers too. Let me work on adding those two papers to the .bib file...

Reply:
This is good 👍🏻

Reply:
Did you want me to add these citations to the .bib and link them up, or are we good to go?

Reply:
If you're referring to each of the citations for each of the algorithm iterations, I think that yes we should do that. If you're referring to "Rob's papers" I'm unsure which ones those are exactly.